coldfront/plugins/qumulo/tests/utils/qumulo_api/test_create_protocol.py

```diff
@@ -5,16 +5,6 @@
 
 @patch("coldfront.plugins.qumulo.utils.qumulo_api.RestClient")
 class CreateProtocol(TestCase):
-    def test_rejects_when_missing_protocol(self, mock_RestClient: MagicMock):
-        qumulo_instance = QumuloAPI()
-
```

Comment:
This test should be kept since the protocol is still mandatory for top-level allocations. 

```
protocols = ["smb"]
```